src/databricks/labs/ucx/aws/access.py

```diff
@@ -266,10 +267,15 @@ def _update_sql_dac_with_instance_profile(self, iam_instance_profile: AWSInstanc
                 f"workspace warehouse config. Do you want UCX to to update it with the uber instance profile?"
             ):
                 return
+        if warehouse_config.security_policy:
+            security_policy = SetWorkspaceWarehouseConfigRequestSecurityPolicy(warehouse_config.security_policy.value)
+        else:
```

Comment:
```suggestion
        if not warehouse_config.security_policy:
```
can it be simplified?